website/content/docs/commands/events.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: events - Command
+description: |-
+  The "events" command interacts with the Vault events subsystem.
+---
+
+# events
+
+The `events` command is used to subscribe to Vault
+[events](/vault/docs/concepts/events) via `events subscribe`. It shows a
+real-time display of events generated by Vault on standard out.
+
+The desired event types (sometimes called "topics") must be specified, which
+may be a glob pattern (with `*` treated as a wildcard).
+
+The output will be JSON objects serialized using the default protobuf JSON
+serialization format, with one line per event received.
+
+Note that this command is designed to run indefinitely. It is similar to
+`tail -f` in the Unix world. This command will not exit on its own unless it
+encounters an unexpected error. As a user, you must terminate this process
+yourself to shut it down.
+
+## Examples
+
+Subscribe to all events:
+
+```shell-session
+$ vault events subscribe '*'
+```
+
+Subscribe to all KV events:
+
+```shell-session
+$ vault events subscribe 'kv*'
+```
+
+Subscribe to all `kv-v2/data-write` events:
+
+```shell-session
+$ vault events subscribe kv-v2/data-write
+```
+
+Subscribe to all non-`data-write` KV events for the secret `secret/data/foo` in
+the current namespace as well as namespace `ns1`:
+
+```shell-session
+$ vault events subscribe -namespaces=ns1 -filter='data_path == secret/data/foo and operation != "data-write"' 'kv*'
+```
+
+## Usage
+
+The following flags are available in addition to the [standard set of
+flags](/vault/docs/commands) included on all commands.
+
+### Options
+
+- `-timeout`: `(duration: "")` - close the WebSocket automatically after the
+  specified duration.
+
+- `-filter` `(string: "")` - Filter expression used to select events to be sent
+  through the WebSocket.
+
+  See [Boundary's docs](/boundary/docs/concepts/filtering) on how these
+  expressions are evaluated and what options are available.
```

Comment:
Why are we sending folks to the Boundary docset?

Reply:
That page is independent of Boundary. They just happen to have written the best documentation on the language implemented by hashicorp/go-bexpr, and there's no sense duplicating it.